{"uid":"b10702f1553b0ae3","name":"test_data_count","fullName":"test_case.test_asset_discovery.test_data_count.TestCase#test_data_count","historyId":"e47756ba57395a677646d37e6a3788cf","time":{"start":1753963418434,"stop":1753963418434,"duration":0},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token","statusTrace":"cls = <class 'test_case.test_asset_discovery.test_data_count.TestCase'>\n\n    @classmethod\n    def setup_class(cls):\n        logger.info('========获取首页数据总量数据，测试开始==========')\n        try:\n            with allure.step('【资产】-【数据列表】接口'):\n                url = f'https://{sc_ip}/apione/v2/data-assets'\n                payload = {\"page_num\":1,\"page_size\":100000}\n                token = getattr(var, \"token\", None)\n                if not token:\n                    logger.error(\"token 获取失败\")\n>                   pytest.fail(\"setup_class: 未获取到 token\")\nE                   Failed: setup_class: 未获取到 token\n\ntest_case\\test_asset_discovery\\test_data_count.py:31: Failed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1753963414831,"stop":1753963415081,"duration":250},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"event_loop_policy","time":{"start":1753963415081,"stop":1753963415081,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"_xunit_setup_class_fixture_TestCase","time":{"start":1753963418437,"stop":1753963418441,"duration":4},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token\n","statusTrace":"  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\python.py\", line 792, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, cls)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\python.py\", line 720, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\pythonProject3\\autotest_api\\test_case\\test_asset_discovery\\test_data_count.py\", line 31, in setup_class\n    pytest.fail(\"setup_class: 未获取到 token\")\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n","steps":[{"name":"【资产】-【数据列表】接口","time":{"start":1753963418437,"stop":1753963418438,"duration":1},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token\n","statusTrace":"  File \"D:\\pythonProject3\\autotest_api\\test_case\\test_asset_discovery\\test_data_count.py\", line 31, in setup_class\n    pytest.fail(\"setup_class: 未获取到 token\")\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":1,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"testStage":{"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token","statusTrace":"cls = <class 'test_case.test_asset_discovery.test_data_count.TestCase'>\n\n    @classmethod\n    def setup_class(cls):\n        logger.info('========获取首页数据总量数据，测试开始==========')\n        try:\n            with allure.step('【资产】-【数据列表】接口'):\n                url = f'https://{sc_ip}/apione/v2/data-assets'\n                payload = {\"page_num\":1,\"page_size\":100000}\n                token = getattr(var, \"token\", None)\n                if not token:\n                    logger.error(\"token 获取失败\")\n>                   pytest.fail(\"setup_class: 未获取到 token\")\nE                   Failed: setup_class: 未获取到 token\n\ntest_case\\test_asset_discovery\\test_data_count.py:31: Failed","steps":[],"attachments":[{"uid":"eb0c1e32ae34329d","name":"log","source":"eb0c1e32ae34329d.txt","type":"text/plain","size":204}],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":1},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"story","value":"【首页】-【数据总量】模块"},{"name":"feature","value":"【首页】-【资产发现】-【数据总量】"},{"name":"parentSuite","value":"test_case.test_asset_discovery"},{"name":"suite","value":"test_data_count"},{"name":"subSuite","value":"TestCase"},{"name":"host","value":"5CD2323XZM"},{"name":"thread","value":"34004-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_case.test_asset_discovery.test_data_count"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"b10702f1553b0ae3.json","parameterValues":[]}